hudi-client/hudi-client-common/src/main/java/org/apache/hudi/client/BaseHoodieWriteClient.java

```diff
@@ -932,6 +947,15 @@ private void startCommit(String instantTime, String actionType, HoodieTableMetaC
             + "table could be in an inconsistent state. Pending restores: " + Arrays.toString(inflightRestoreTimeline.getInstantsAsStream()
             .map(instant -> instant.getTimestamp()).collect(Collectors.toList()).toArray()));
 
+    HoodieInstant requestedInstant = new HoodieInstant(State.REQUESTED, instantTime, actionType);
+    this.txnManager.beginTransaction(Option.of(requestedInstant),
+        lastCompletedTxnAndMetadata.isPresent() ? Option.of(lastCompletedTxnAndMetadata.get().getLeft()) : Option.empty());
```

Comment:
The last txn metadata may be useless for the time sequence check, we just need the lock, but's it should be fine.